.gitignore

```diff
@@ -11,7 +11,6 @@ build
 node_modules
 
 # config
-server/config.ts
```

Comment:
Please revert this change, It's safer so it doesn't accidentally gets pushed to git

Reply:
The idea was to make credential management for both build types (npm & docker) to happen through .env file (in .gitignore). Server config.ts should be committed to the repository as it only takes values from env and user shouldn't modify it directly.

Reply:
I think we should have both, `config.ts` and `.env`, because changing it to .env will break for all the users that have the config.ts. This might create confusion with the current guides too

Reply:
Alright. I'm not that familiar with nodejs, but would it be possible to create abstraction config layer between config.ts and .env. Meaning that the file could be optional and it would read values from there if .env values are missing. I know that TypeScript starts screaming in case file is missing.

Reply:
I don't think that's possible.